.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - staging
```

Comment:
I'm thinking if we should add this to all PRs.... I dont think this will take a lot of time/resources.
If we create an epic branch with a lot of PRs, we could potentially lose a lot of time if we find out that there's an issue when merging to stg. But let's see how long it takes first. what d u think?